src/Twigc/DefaultCommand.php

```diff
@@ -349,6 +359,10 @@ public function doRender(InputInterface $input, OutputInterface $output) {
 			}
 		}
 
+		if ($wantEnv) {
```

Comment:
Can you put spaces around the condition (inside the parens) to match the formatting used elsewhere? Also see above about checking the value directly